code/frontend/src/app/_services/user.service.ts

```diff
@@ -22,7 +22,7 @@ export class UserService {
   }
 
   create(user: User) {
-    return this.http.post('/api/v1/persons', user).pipe(map((response: Object) => response));
+    return this.http.post('/api/v1/persons/', user).pipe(map((response: Object) => response));
```

Comment:
ciekawa zmiana, skąd sie wziela?

Reply:
To było mi potrzebne do wygenerowania błędu, ale zrobiłem commit razem z popsutą wersją i później poprawiłem na poprawą wersję. Tylko teraz się zastanawiam czy jest ok, bo jak patrzę w historii to wcześniej było bez '/' . Teraz jak testowałem to działa dodawanie